pkg/rollupsmachine/io.go

```diff
@@ -0,0 +1,123 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package rollupsmachine
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type Input struct {
+	ChainId        uint64
+	AppContract    [20]byte
```

Comment:
Because this is a `/pkg` module. It should not reference things from `/internal`.
Nevertheless, I've changed this to use the new `Address` type alias (defined alongside `Cycle`, `Output`, etc.).